### DISCUSSION TITLE
Extra user permissions level for scanning

### DISCUSSION THREAD

**Discussion by justynpride** [HiEventsDev/Hi.Events#108] [Ideas]
I'm sure this must be on the road map, but I'd love to see an extra permission level for a user that allows them to access the check in only feature for an event. Ideally you could specify which event and not just check-in. I might have different team doing check in for different events, some are volunteers so the more that access can be restricted the better.

**Comment by daveearley** [comment]
That's a great idea! We'll have "Check-In lists" soon. The lists can apply to all, or specific tickets. You'll be able to share a link to the list with volunteers etc. and it won't give them access to the rest of the app.

**Reply by justynpride** [reply]
Wonderful! You've created a great app.
